scripts/integration-tests/publish-local.sh

```diff
@@ -25,6 +25,17 @@ initializeE2Egit
 
 yarn
 
+# Does not trap on error
+node scripts/integration-tests/utils/e2e-check.js && :; ret=$?
```

Comment:
Can we just run this script with `--experimental-fetch`? So we don't have to install `node-fetch` at all. The flag is available in Node 18+

Reply:
It seems that some CIs are running on NODE16, and I think `node-fetch` may also be used in the future.

Reply:
Thanks!